lm_eval/tasks/hendrycks_test.py

```diff
@@ -126,41 +125,47 @@ def validation_docs(self):
     def test_docs(self):
         return map(self._process_doc, self.dataset["test"])
 
+    def fewshot_context(self, doc, num_fewshot, **kwargs):
+        subject = self.DATASET_NAME
+        description = f"The following are multiple choice questions (with answers) about {subject}."
```

Comment:
"(with answers)" does not appear to be in the paper:

<img width="352" alt="Screen Shot 2023-05-24 at 1 51 34 PM" src="https://github.com/EleutherAI/lm-evaluation-harness/assets/15899312/1abe9d76-13c2-4548-83c2-abaaf9e73965">

Reply:
It looks like “with answers” is not specified in the paper but is specified in the [official implementation](https://github.com/hendrycks/test/blob/4450500f923c49f1fb1dd3d99108a0bd9717b660/evaluate.py#L39)

Reply:
Yep, that was the source that I've used as the reference.